addon-test-support/setup-qunit.ts

```diff
@@ -0,0 +1,9 @@
+import * as QUnit from 'qunit';
+
+export function setupQUnitA11yAudit(qunit: QUnit) {
```

Comment:
Minor suggestion on the naming of this function. It's not apparently clear that this adds a UI in the QUnit test runner and implies that it's somehow directly related to the `a11yAudit` function and is maybe a requisite for invocation. I think this ambiguity can be cleared up by naming it `setupQUnitA11yAuditToggle` or `setupQUnitA11yAuditCheckbox`. Something along those lines.